app/scripts/modules/kubernetes/src/v2/pipelines/stages/runJob/RunJobExecutionDetails.tsx

```diff
@@ -1,43 +1,131 @@
 import * as React from 'react';
 import { get } from 'lodash';
 
-import {
+import { IExecutionDetailsSectionProps, ExecutionDetailsSection, AccountTag, IManifest } from '@spinnaker/core';
+
+import { JobManifestPodLogs } from '../deployManifest/react/JobManifestPodLogs';
+
+import { KubernetesManifestService } from 'kubernetes/v2/manifest/manifest.service';
+import { IManifestSubscription } from '../deployManifest/react/DeployStatus';
+
+interface IStageManifest {
+  kind: string;
+  apiVersion: string;
+  metadata: {
+    namespace: string;
+    name: string;
+  };
+}
+
+interface IStageDeployedJobs {
+  [namespace: string]: string[];
+}
+
+interface IRunJobExecutionDetailsState {
+  subscription: IManifestSubscription;
+  manifestId: string;
+}
+
+export class RunJobExecutionDetails extends React.Component<
   IExecutionDetailsSectionProps,
-  ExecutionDetailsSection,
-  AccountTag,
-  ReactModal,
-  ReactInjector,
-  LogsModal,
-  ILogsModalProps,
-} from '@spinnaker/core';
-
-export class RunJobExecutionDetails extends React.Component<IExecutionDetailsSectionProps> {
+  IRunJobExecutionDetailsState
+> {
   public static title = 'runJobConfig';
 
-  public showLogsModal = (_event: any): void => {
-    const { stage, execution } = this.props;
-    const { executionService } = ReactInjector;
-    executionService.getExecution(execution.id).then((fullExecution: any) => {
-      const fullStage = fullExecution.stages.find((s: any) => s.id === stage.id);
-      if (!fullStage) {
-        return;
-      }
-
-      const modalProps = { dialogClassName: 'modal-lg modal-fullscreen' };
-      ReactModal.show(
-        LogsModal,
-        {
-          logs: get(fullStage, 'context.jobStatus.logs', 'No log output found.'),
-        } as ILogsModalProps,
-        modalProps,
-      );
+  constructor(props: IExecutionDetailsSectionProps) {
```

Comment:
can replace the constructor with the following since `props` is not used to initialize state:
```suggestion
    public state = { subscription: { id: '', unsubscribe: null, manifest: null }, manifestId: '' };
```